esmf_regrid/tests/unit/experimental/unstructured_scheme/test_regrid_rectilinear_to_unstructured.py

```diff
@@ -233,3 +233,36 @@ def test_mask_handling():
     assert ma.allclose(expected_0, result_0.data)
     assert ma.allclose(expected_05, result_05.data)
     assert ma.allclose(expected_1, result_1.data)
+
+
+def test_resolution():
+    """
+    Basic test for :func:`esmf_regrid.experimental.unstructured_scheme.regrid_rectilinear_to_unstructured`.
+
+    Tests the resolution keyword with grids that would otherwise not work.
```

Comment:
This would involve causing an error within ESMF, which feels like it would be out of scope for our tests.

Reply:
Yes I can see that point. It's good that you at least mentioned "otherwise not work", as it allowed me to test this myself and see the ESMF error.

Reply:
I've decided to add a comment on what the alternative behaviour would be rather than a test. Testing to ensure failures seems like the wrong way to do things.